prusti-common/src/vir/ast/expr_transformers.rs

```diff
@@ -707,7 +727,21 @@ pub trait FallibleExprFolder: Sized {
         ))
     }
     */
-    fn fallible_fold_inhale_exhale(
+    fn fallible_fold_backend_func_app(
+        &mut self,
+        func: BackendFunc,
+        args: Vec<Expr>,
+        pos: Position,
+    ) -> Result<Expr, Self::Error> {
+        Ok(Expr::BackendFuncApp(
+            func,
+            args.into_iter()
+                .map(|e| self.fallible_fold(e))
+                .collect::<Result<Vec<_>, Self::Error>>()?,
+            pos
+        ))
+    }
+    fn fallible_inhale_exhale(
```

Comment:
```suggestion
    fn fallible_fold_inhale_exhale(
```
Then also undo the change on line 833. It seems a rebase accidentally renamed this function when it should not have.